examples/axes_grid1/inset_locator_demo.py

```diff
@@ -6,11 +6,11 @@
 """
 
 ###############################################################################
-# The `.inset_locator`'s `~.axes_grid1.inset_axes` allows to easily place
-# insets in the corners of the axes by specifying a width and height and
-# optionally a location (loc) which accepts locations as codes, similar to
-# `~matplotlib.axes.Axes.legend`.
-# By default, the inset is offset by some points from the axes - this is
+# The `.inset_locator`'s `~mpl_toolkits.axes_grid1.inset_axes` allows
```

Comment:
If I'm not mistaken it should be 
`~mpl_toolkits.axes_grid1.inset_locator.inset_axes`, which may probably be shortened to `~inset_locator.inset_axes`. So I guess I was the one to actually make this mistake when I last edited that example.